packages/editor/src/components/block-mover/index.js

```diff
@@ -66,6 +66,30 @@ export class BlockMover extends Component {
 					onFocus={ this.onFocus }
 					onBlur={ this.onBlur }
 				/>
+				<Draggable
+					elementId={ blockElementId }
+					transferData={ {
+						type: 'block',
+						fromIndex: order,
+						rootClientId,
+						clientId: clientIds,
+						layout,
+					} }
+				>
+					{
+						( { onDraggableStart, onDraggableEnd } ) => (
+							<IconButton
```

Comment:
Not sure we should use the button styles here. It looks a bit weird on hover.

Reply:
I think the space should also be a bit tighter overall.

Reply:
I'll leave @jasmussen apply his svg / css :magician: skills while I watch from the distance and learn.